nixos/modules/services/web-apps/wiki-js.nix

```diff
@@ -113,7 +113,14 @@ in {
       documentation = [ "https://docs.requarks.io/" ];
       wantedBy = [ "multi-user.target" ];
 
-      path = with pkgs; [ coreutils ];
+      path = with pkgs; [
+        coreutils
```

Comment:
Isn't `coreutils` already in the default path for systemd services?

Regardless, this change looks reasonable enough to me

Reply:
Yep, that looks correct. I took out coreutils and all the tests still pass, and manual testing seems fine on my systems. Updated.